styles/styles.css

```
@@ -671,21 +671,24 @@ body {
   color: #ffffff;
 }
 
-.breadcrumbs-item:hover .breadcrumbs-item-object {
+.breadcrumbs-link{
```

```suggestion
.breadcrumbs-link {
```
старайся следить за чистотой кода заранее. а то на защите завернут)
(исправь пожалуйста везде где такое есть)